presidio-analyzer/presidio_analyzer/nlp_engine/nlp_artifacts.py

```diff
@@ -18,15 +26,17 @@ def __init__(
         tokens: Doc,
         tokens_indices: List[int],
         lemmas: List[str],
-        nlp_engine,  # noqa ANN001
+        nlp_engine: "NlpEngine",  # noqa F821
         language: str,
+        scores: Optional[List[float]] = None,
     ):
         self.entities = entities
         self.tokens = tokens
         self.lemmas = lemmas
         self.tokens_indices = tokens_indices
         self.keywords = self.set_keywords(nlp_engine, lemmas, language)
         self.nlp_engine = nlp_engine
+        self.scores = scores if scores else [0.85] * len(entities)
```

Comment:
Why auto assign 0.85 by default if no scores are available?

Reply:
Understood. Is this something that should be revisited from a whole Presidio level? Perhaps I am just lacking context, but returning 0.85 as a fixed default value seems misleading to me.

No need to address in this PR.

Reply:
We can definitely consider changing this. It was like they even before my time 🙂

Reply:
Create a new issue to revisit this in the future: https://github.com/microsoft/presidio/issues/1190